src/components/ChoiceList/ChoiceList.tsx

```diff
@@ -59,9 +59,7 @@ function ChoiceList({
   error,
   name = getUniqueID(),
 }: CombinedProps) {
-  const ControlComponent: typeof Checkbox | typeof RadioButton = allowMultiple
-    ? Checkbox
-    : RadioButton;
+  const ControlComponent: any = allowMultiple ? Checkbox : RadioButton;
```

Comment:
Something like this should work

```tsx
type CheckboxComponent = React.ComponentClass<Props, State> & typeof Checkbox;
type RadioComponent = React.SFC<Props> & typeof RadioButton;
const ControlComponent: CheckboxComponent | RadioComponent = allowMultiple ? Checkbox : RadioButton;
```

Reply:
No dice, for the same reasons as above. https://github.com/Microsoft/TypeScript/issues/28768 breaks us, but it's no longer an error in newer versions of TS.

Reply:
I tried it locally yesterday and it seemed to work but I may have had a different version of TS locally (3.4 is my guess). Same comment as above for this 👌